keps/NNNN-kep-template/README.md

```diff
@@ -270,6 +266,55 @@ when drafting this test plan.
 [testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
 -->
 
+[ ] I/we understand the owners of the involved components may require updating
```

Comment:
I didn't fully changed that because I think the change wasn't reflecting what I wanted.

I wanted to say that "component OWNERS" (say approvers) have a right to request additional tests before proceeding. Does that make sense?